modules/dnn/src/layers/einsum_layer.cpp

```diff
@@ -671,8 +861,6 @@ void LayerEinsumImpl::createOutputSubsctipt()
             {
                 CV_Error(Error::StsError,
                 "Provided output subscript does not include ellipsis while Inputs subscrits constain ellipsis");
-            } else {
```

Comment:
The function still does nothing, but check state. It does not much the function name.

Reply:
lets rename it